### DISCUSSION TITLE
An error occurred!

### DISCUSSION THREAD

**Discussion by Xshafi** [Sketchware-Pro/Sketchware-Pro#823] [Q&A]
Dear Developers
![Screenshot_2023-04-01-07-06-01](https://user-images.githubusercontent.com/95597052/229260482-d99a2427-19bc-4bcd-85a2-733d9225d633.png)
,

I am facing an issue with exporting my app created on Sketchwere. I am getting an error message and the app is not exporting. Can we please discuss this and find a solution to the problem?

Thank you.

**Comment by j-px** [comment]
Something's wrong in main.xml at line 8. You probably added invalid text to some Widget's inject property.
